src/main/java/org/jenkinsci/test/acceptance/po/WorkflowJob.java

```diff
@@ -37,4 +47,20 @@ public WorkflowJob(Injector injector, URL url, String name) {
     public final Control script = control("/definition/script");
     public final Control sandbox = control("/definition/sandbox");
 
+    public String copyResourceStep(String filePath) {
```

Comment:
Oracle coding style presumes that unmarked parameters are always Nonnull. BTW FindBugs does not work in such way by default, hence it won't discover the issue if it happens. It's NIT for the test suite, but nice to have